tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -896,7 +944,7 @@ async def process_bidi_event(method, params):
                 # TODO: what to do if there are more then 1 unexpected exceptions?
                 raise unexpected_exceptions[0]
 
-            test_driver_message = self._get_next_message(protocol, url, test_window)
+            test_driver_message = protocol.testdriver.get_next_message(url, test_window)
             self.logger.debug("Receive message from testdriver: %s" % test_driver_message)
```

Comment:
Not related, looks like at L942 we are blindly trying the testdriver loop, but in reality only a small portion of test harness tests use test driver. Is there any chance for optimization here?

Reply:
OK, look like that is needed to get the test results.